apps/menu-bar/package.json

```diff
@@ -19,25 +19,27 @@
   },
   "dependencies": {
     "@apollo/client": "^3.8.1",
-    "@expo/metro-config": "~0.18.11",
-    "@expo/metro-runtime": "~3.2.3",
+    "@expo/metro-runtime": "~4.0.0",
     "@expo/styleguide-native": "^1.0.1",
-    "@fluentui/react-components": "^9.46.4",
+    "@fluentui/react-checkbox": "^9.2.40",
     "@fluentui/react-icons": "^2.0.227",
+    "@fluentui/react-progress": "^9.1.90",
+    "@fluentui/react-provider": "^9.18.0",
+    "@fluentui/react-switch": "^9.1.97",
+    "@fluentui/react-theme": "^9.1.22",
```

Comment:
latest release of `@fluentui/react-components` is currently broken so we need to use the individual packages directly